geocodingv7/geocoding.go

```diff
@@ -0,0 +1,88 @@
+package geocodingv7
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Geocoding allows forward geocode addresses and geo-positions.
```

Comment:
The first line of the comment should ideally describe what the function does/will return?